.tekton/integration-tests/pipelines/lightspeed-operator-upgrade-e2e-test-pipeline-418.yaml

```diff
@@ -272,7 +272,8 @@ spec:
               git config --global user.name olsci
               git fetch origin ${COMMIT_SHA}
               git checkout ${COMMIT_SHA}
-              export BUNDLE_IMAGE = $(cat lightspeed-catalog-4.18/index.yaml | yq '.relatedImages[] | select(.name == "lightspeed-operator-bundle") | .image')
+              export BUNDLE_IMAGE=$(cat lightspeed-catalog-4.19/index.yaml | yq '.relatedImages[] | select(.name == "lightspeed-operator-bundle") | .image')
```

Comment:
should it be `lightspeed-catalog-4.18` ?

Reply:
100% it should, fixed
thank you!